gofakes3_test.go

```diff
@@ -227,6 +227,7 @@ func TestCreateObjectWithInvalidContentLength(t *testing.T) {
 		t.Fatal(rs.StatusCode, "!=", http.StatusBadRequest)
 	}
 }
+
```

Comment:
```suggestion
```

Reply:
Let's not add formatting related changes to this PR

Reply:
done.